bitbots_localization/include/bitbots_localization/localization.h

```diff
@@ -219,8 +219,8 @@ class Localization {
 
   void getMotion();
 
-  geometry_msgs::Vector3 movement_;
-  geometry_msgs::Vector3 movement2_;
+  geometry_msgs::Vector3 cartesian_movement_;
```

Comment:
As in #43, I woult prefer linear instead of cartesian

Reply:
I know. #43 gets modified first. I just wanted to build this one on top of it and wait (draft) until the other one is finished.

Reply:
alright

Reply:
Done